assume/markets/clearing_algorithms/network_clear.py

```diff
@@ -0,0 +1,375 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import logging
+import os
+import subprocess
+import warnings
+from collections import Counter
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import numpy as np
+import pandapower as pp
+import pandas as pd
+import pypsa
+from shapely.errors import ShapelyDeprecationWarning
+
+warnings.filterwarnings("ignore", category=ShapelyDeprecationWarning)
+warnings.filterwarnings("ignore", category=FutureWarning)
```

Comment:
Why is this needed? Can't you install a recent version instead?
I don't like filtering warnings a lot

Reply:
sorry, this whole file was just a draft, sorry you read it, redispatch.py is the main file